agent/agent.sh

```diff
@@ -1,23 +1,29 @@
 #!/bin/sh
 
-set -e
+function signal {
+  cfn-signal -e $1 --stack $STACK --resource ASG --region $REGION
+  [ $? -ne 0 ] && echo "i2kit agent: cfn-signal failed to be sent."
+  [ $? -eq 0 ] && echo "i2kit agent: cfn-signal successfully sent."
+}
 
-[ -z "$COMPOSE" ] && echo "'COMPOSE' must be defined" && return 1
-[ -z "$STACK" ] && echo "'STACK' must be defined" && return 1
-[ -z "$REGION" ] && echo "'REGION' must be defined" && return 1
+[ -z "$COMPOSE" ] && echo "i2kit agent: 'COMPOSE' must be defined." && return 1
```

Comment:
I'd create a `log` function to append the i2kit agent prefix once instead of on every log.